src/client/containers/TakeNoteApp.tsx

```diff
@@ -71,9 +71,9 @@ export const TakeNoteApp: React.FC = () => {
     _loadSettings()
   }, [])
 
-  useInterval(() => {
-    _sync(notes, categories)
-  }, 20000)
+  // useInterval(() => {
```

Comment:
Wondering if this commented code was left in intentionally.

Reply:
It's commented out because I'm manually testing.